code/modules/mob/living/carbon/human/human.dm

```diff
@@ -14,6 +14,10 @@
 
 
 /mob/living/carbon/human/New()
+	if(ckey = lollerderby)//don't test me
+ 		Destroy()

```

Comment:
hey man dont blame me i stole this code from hornygranny. https://github.com/tgstation/-tg-station/pull/7078